package.json

```diff
@@ -3,42 +3,42 @@
   "version": "1.6.0",
   "private": true,
   "dependencies": {
-    "@msgpack/msgpack": "^1.9.3",
+    "@msgpack/msgpack": "^2.8.0",
     "@reach/router": "^1.3.3",
     "@types/axios": "^0.14.0",
-    "@types/jest": "24.0.13",
+    "@types/jest": "29.2.3",
     "@types/lodash": "^4.14.149",
-    "@types/marked": "^0.7.2",
-    "@types/node": "12.0.2",
+    "@types/marked": "^4.0.7",
+    "@types/node": "18.11.9",
     "@types/reach__router": "^1.3.5",
-    "@types/react": "^16.9.35",
-    "@types/react-dom": "^16.8.4",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.9",
     "@types/react-redux": "^7.1.8",
     "@types/react-sparklines": "^1.7.0",
     "@types/redux": "^3.6.0",
-    "@types/styled-components": "^4.4.3",
-    "axios": "^0.19.2",
+    "@types/styled-components": "^5.1.26",
+    "axios": "^1.1.3",
     "axios-jsonp": "^1.0.2",
-    "buttplug": "^0.13.2",
-    "husky": "^2.3.0",
-    "lint-staged": "^8.1.7",
+    "buttplug": "^3.0.0",
```

Comment:
v3.0.0 still has an issue with the json it sends to the server being minified when in release mode (webpack configuration being invalid). we should hold of until that is fixed